conda_build/build.py

```diff
@@ -1441,13 +1444,14 @@ def create_info_files(m, replacements, files, prefix):
 
 
 def get_short_path(m, target_file):
+    if m.python_version_independent:
+        if target_file.find("site-packages") >= 0:
+            return target_file[target_file.find("site-packages") :]
     if m.noarch == "python":
         entry_point_script_names = get_entry_point_script_names(
             m.get_value("build/entry_points")
         )
-        if target_file.find("site-packages") >= 0:
-            return target_file[target_file.find("site-packages") :]
-        elif target_file.startswith("bin") and (
+        if target_file.startswith("bin") and (
```

Comment:
Should this include the slash to prevent matches with paths like `binocular.dat`?

```suggestion
        if target_file.startswith("bin/") and (
```

(And in that case, update the replace below?)

Reply:
Let's leave it for a different PR?